code/modules/religion/burdened/burdened_trauma.dm

```diff
@@ -143,13 +143,33 @@
 	if(special) //aheals
 		return
 
-	if(istype(old_organ, /obj/item/organ/internal/eyes))
+	/// only organs that are slotted in these count. because there's a lot of useless organs to cheese with.
+	var/list/slots_youd_actually_miss = list(
+		ORGAN_SLOT_BRAIN,
+		ORGAN_SLOT_HEART,
+		ORGAN_SLOT_LUNGS,
+		ORGAN_SLOT_EYES,
+		ORGAN_SLOT_EARS,
+		ORGAN_SLOT_TONGUE,
+		ORGAN_SLOT_LIVER,
+		ORGAN_SLOT_STOMACH,
+	)
+	var/list/bad_traits = list(
+		TRAIT_NOHUNGER = ORGAN_SLOT_STOMACH,
+		TRAIT_NOBREATH = ORGAN_SLOT_LUNGS,
+		TRAIT_NOMETABOLISM = ORGAN_SLOT_LIVER,
```

Comment:
what happens if you get these traits AFTER you've already removed these organs? plasmamen, for example, get their NOHUNGER trait from their lungs- would removing their lungs before their stomach give more points than the other way around?

Reply:
If you have the traits, the organ slots simply don't work with the system. In theory, there could be an exploit if you were able to repeatedly remove the organ, add the trait, remove the trait, and remove the organ again, but... do we even have a way to gain and remove these traits controllably and repeatably?

Reply:
> If you have the traits, the organ slots simply don't work with the system. In theory, there could be an exploit if you were able to repeatedly remove the organ, add the trait, remove the trait, and remove the organ again, but... do we even have a way to gain and remove these traits controllably in a repeatable situation?

wouldn't removing/readding plasmaman lungs allow you to manipulate the nohunger trait independently of your stomach?